components/camel-jira/src/main/java/org/apache/camel/component/jira/consumer/AbstractJiraConsumer.java

```diff
@@ -47,7 +47,7 @@ public AbstractJiraConsumer(JiraEndpoint endpoint, Processor processor) {
 
     // Ignore maxResults if it's <= 0.
     protected List<Issue> getIssues(String jql, int start, int maxPerQuery, int maxResults) {
-        LOG.info("Indexing current JIRA issues...");
+        LOG.info("Start indexing current JIRA issues...");
```

Comment:
Do we really need this INFO logging noise? If this method is invoked frequently you get spammed with logs. Can we lower this to DEBUG

Reply:
Done. Thanks for the review.